src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -613,6 +613,26 @@ End Module]]></Document>)
             End Using
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
Yes, I was going to propose that too.  In the long term, maybe we make another overload that doesn't require the int param.